apps/main/next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
What about the NOTE ser

Reply:
This happened automatically, tried to revert but nextjs automatically does this again